lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -343,15 +355,28 @@ def test_masquerade_as_specific_user_on_self_paced(self):
         u'fôô@bar',  # Unicode username with @, which is what the ENABLE_UNICODE_USERNAME feature allows
     )
     @patch.dict('django.conf.settings.FEATURES', {'DISABLE_START_DATES': False})
+    @patch.dict("django.conf.settings.FEATURES", {'ENABLE_PREREQUISITE_COURSES': True})
     def test_masquerade_as_specific_student(self, username):
         """
         Test masquerading as a specific user.
 
         We answer the problem in our test course as the student and as staff user, and we use the
         progress as a proxy to determine who's state we currently see.
         """
+        course_key = CourseKey.from_string(self.course.id)
```

Comment:
Course.id is a `CourseKey` object.

```suggestion
        course_key = self.course.id
```